src/cryptoadvance/spectrum/elsock.py

```diff
@@ -109,6 +116,12 @@ def __init__(
         # therefore setting it at the very end:
         self._on_recreation_callback = socket_recreation_callback
 
+    def shutdown(self):
```

Comment:
Those are not used, yet, correct? Perhaps mark with a comment.

Reply:
Good catch! I did a `git reset` and lost some changes.